### DISCUSSION TITLE
Export transport orders to the openTCS 6.0.0 OperationsDesk

### DISCUSSION THREAD

**Discussion by antichecker** [openTCS/opentcs#155] [General]
I'm looking for a way to create transport orders outside of openTCS and then give them over to the OperationsDesk (or Kernel, if that is where they should go) to be put in the transport orders queue. 
The idea is that I want to extract transport orders from a 'discrete event simulation'-program (WITNESS), translate them into something openTCS understands and then give them over to the OperationsDesk to be dispatched. This should preferably happen locally on Windows 10.
Does openTCS 6.0.0 support this kind of interaction? How might I be able to do it?

**Comment by martingr** [comment]
For this purpose, you may want to use the web API that the openTCS kernel provides (see [1]), which provides an endpoint for creating transport orders. Like transport order created directly in/via the Operations Desk application, transport orders created via the web API will be added to the kernel's transport order pool and displayed in the Operations Desk application.

[1]: https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openTCS/opentcs/v6.0.0/openTCS-Documentation/src/docs/service-web-api-v1/openapi.yaml

**Comment by antichecker** [comment]
Thank you for the answer!
